pythonforandroid/recipes/scikit-learn/__init__.py

```diff
@@ -0,0 +1,40 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe, Recipe
+from multiprocessing import cpu_count
+
+
+class ThisRecipe(CompiledComponentsPythonRecipe):
```

Comment:
Can you follow the naming of the recipe class like the ones we already have on `python-for-android` ?

E.g. `ScikitLearnRecipe`